src/server/HelixProject.js

```diff
@@ -169,6 +171,30 @@ export default class HelixProject {
     }
   }
 
+  async initWatches() {
+    const pkgFile = path.resolve(this.directory, 'package.json');
+    if (fs.existsSync(pkgFile)) {
+      const pkg = fs.readFileSync(pkgFile);
+      const pkgObj = JSON.parse(pkg);
+      const { watches } = pkgObj;
+      if (watches !== undefined) {
+        this.log.debug(`[Glob-Watcher]Registering watches: ${JSON.stringify(watches, null, 2)}`);
+        const watchMap = new Map();
+        watches.forEach((watch) => {
+          if (watch.globs !== undefined && watch.command !== undefined) {
```

Comment:
This can probably be simplified by not having globs.

Reply:
Looking at [chokidar documentation](https://github.com/paulmillr/chokidar), I couldn't find a way to specify file extensions other than through globs. And I think we need to filter by file extension to avoid unnecessary command executions.

Reply:
Hm. Then 

```json
{
  "scripts": {
    "watch:*.ts": "tscompile",
    "watch:*.less": "lessc"
  }
}
```

could work. One watcher, and the dispatching works based on the glob.

And I'd implement it lazily: no glob support first, and when we hear that it's being used (and slow) we add glob support.